_posts/2024-08-29-socratic-seminar-028.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+type: socratic
+title: "Seminário Socrático 028"
+meetup: https://www.meetup.com/clubebitcoinsp/events/302289804/
+---
+
+## Avisos
+
+- Entrem no grupo do Whatsapp [São Paulo Bitdevs](https://chat.whatsapp.com/HiaPqjmUqER5djFPR1Yl3T)!
+- Respeite a privacidade dos participantes.
+- Os meetups nunca são gravados. Queremos todos a vontade para participar e discutir os assuntos programados, de forma anônima se assim o desejarem.
+
+## Agradecimentos
+
+- Agradecemos à Vinteum pela casa, comidas e bebidas.
+
+## Cronograma
+
+### Aquecimento
+- Enigma
+- [Bitaxe achou um bloco](https://mempool.space/tx/fe75e7d6e9f1c73f77c3dbeddd911dc6fbb8bcf98b7a37fec1820adba989e378)
+- [Apple App Store censurando aplicativos de VPN](https://x.com/protonvpn/status/1825181045035074016)
+- [Mutiny wallet foi de arrasta pra cima](https://blog.mutinywallet.com/mutiny-wallet-is-shutting-down/)
+- [SimpleX Chat v6.0.0](https://www.nobsbitcoin.com/simplex-chat-v6-0/)
+- [Vírus trocando endereços na área de transferência](https://x.com/SandLabs_21/status/1820311557269594463)
+
+### Bitcoin
+- [Floresta v0.6.0](https://medium.com/vinteum-org/floresta-update-simplifying-bitcoin-node-integration-for-wallets-6886ea7c975c)
+- [Mempool v3.0.0](https://x.com/mempool/status/1828057401968533629)
+- [BDK 1.0.0-beta.1](https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-beta.1)
+- [Existe incentivo econômico em usar P2WSH ao invés de P2TR?](https://bitcoin.stackexchange.com/questions/123500/is-there-an-economic-incentive-to-use-p2wsh-over-p2tr)
+- [Bitcoin Core's conservative next block fee recommendations](https://x.com/mononautical/status/1826679495568564677) [2](https://x.com/mononautical/status/1826604180251050388)
+- [BitVM2](https://bitvm.org/bitvm_bridge.pdf)
+- [New time warp vulnerability in testnet4](https://delvingbitcoin.org/t/zawy-s-alternating-timestamp-attack/1062) [2](https://bitcoinops.org/en/newsletters/2024/08/16/)
+
+### Lightning
+- [Bolt11 em C# / .NET](https://x.com/ngoline/status/1828193300546724236)
+- [Zeus v0.9.0](https://github.com/ZeusLN/zeus/releases/tag/v0.9.0)
+
+### Criptografia
+- [Engenharia de uma carteira de bitcoin com backdoor](https://www.usenix.org/system/files/woot24-scott.pdf)
+- [Dark Skippy](https://darkskippy.com/) e [mitigações](https://x.com/nunchuk_io/status/1820710359780704557)
```

Comment:
```suggestion
### Aquecimento

- Enigma
- [Bitaxe achou um bloco](https://mempool.space/tx/fe75e7d6e9f1c73f77c3dbeddd911dc6fbb8bcf98b7a37fec1820adba989e378)
- [Apple App Store censurando aplicativos de VPN](https://x.com/protonvpn/status/1825181045035074016)
- [Mutiny wallet foi de arrasta pra cima](https://blog.mutinywallet.com/mutiny-wallet-is-shutting-down/)
- [SimpleX Chat v6.0.0](https://www.nobsbitcoin.com/simplex-chat-v6-0/)
- [Vírus trocando endereços na área de transferência](https://x.com/SandLabs_21/status/1820311557269594463)

### Bitcoin

- [Floresta v0.6.0](https://medium.com/vinteum-org/floresta-update-simplifying-bitcoin-node-integration-for-wallets-6886ea7c975c)
- [Mempool v3.0.0](https://x.com/mempool/status/1828057401968533629)
- [BDK 1.0.0-beta.1](https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-beta.1)
- [Existe incentivo econômico em usar P2WSH ao invés de P2TR?](https://bitcoin.stackexchange.com/questions/123500/is-there-an-economic-incentive-to-use-p2wsh-over-p2tr)
- [Bitcoin Core's conservative next block fee recommendations](https://x.com/mononautical/status/1826679495568564677) [2](https://x.com/mononautical/status/1826604180251050388)
- [BitVM2](https://bitvm.org/bitvm_bridge.pdf)
- [New time warp vulnerability in testnet4](https://delvingbitcoin.org/t/zawy-s-alternating-timestamp-attack/1062) [2](https://bitcoinops.org/en/newsletters/2024/08/16/)

### Lightning

- [Bolt11 em C# / .NET](https://x.com/ngoline/status/1828193300546724236)
- [Zeus v0.9.0](https://github.com/ZeusLN/zeus/releases/tag/v0.9.0)

### Criptografia

- [Engenharia de uma carteira de bitcoin com backdoor](https://www.usenix.org/system/files/woot24-scott.pdf)
- [Dark Skippy](https://darkskippy.com/) e [mitigações](https://x.com/nunchuk_io/status/1820710359780704557)
```

Consistency with H2 please

Reply:
fixed in 6d0cbfe3cf3ceb471d1b72cd33af4c3db4aa6672 ty